tensorflow/contrib/keras/python/keras/utils/generic_utils.py

```diff
@@ -195,7 +195,7 @@ def func_dump(func):
   Returns:
       A tuple `(code, defaults, closure)`.
   """
-  code = marshal.dumps(func.__code__).decode('raw_unicode_escape')
+  code = marshal.dumps(func.__code__).replace(b'\\',b'/').decode('raw_unicode_escape')
```

Comment:
What's going on here? It doesn't seem generally safe to do this, but I'm not familiar with this code...